cli/pcluster/config/validators.py

```diff
@@ -1066,6 +1067,15 @@ def compute_instance_type_validator(param_key, param_value, pcluster_config):
     else:
         errors, warnings = ec2_instance_type_validator(param_key, param_value, pcluster_config)
 
+        instance_nics = get_instance_network_interfaces(param_value)
```

Comment:
general question: have we checked how many times we call the describe_instance_types? is this cached anywhere?

Reply:
It's not cached right now and I think it's a good idea caching it.